src/dowel/csv_output.py

```diff
@@ -41,13 +44,8 @@ def record(self, data, prefix=''):
                     extrasaction='ignore')
                 self._writer.writeheader()
 
-            if to_csv.keys() != self._fieldnames:
-                self._warn('Inconsistent TabularInput keys detected. '
```

Comment:
Since this is the last call to `_warn`, and `_warn` is a private method, can you delete that and other unused fields?

Reply:
Of course!